### PR TITLE
104 x dev nano sos plotter tidy

### PR DESCRIPTION
**Pull request by ipapaver**
Updates on the Fake Rate MC framework.
Fixes in the 3l semi dd bkg mca files.
Deleting unnecessary files.
Changing the DY fakes matching in 3l mca files --> adding mcPromptGamma ==0 requirement in order to avoid double counting the conversions

### CODE REVIEW

File: TTHAnalysis/python/plotter/mcPlots.py [hunk 1]
```diff
@@ -990,7 +990,7 @@ def printOnePlot(self,mca,pspec,pmapIn,mytotal=None,makeCanvas=True,outputDir=No
                     signorms = doStackSignalNorm(pspec,pmap,options.showIndivSigShapes or options.showIndivSigs,extrascale=options.signalPlotScale, norm=not options.showIndivSigs)
                     for signorm in signorms:
                         if outputDir: 
-                            signorm.SetDirectory(outputDir); outputDir.WriteTObject(signorm)
+                            signorm.SetDirectory(outputDir); #outputDir.WriteObjectAny(signorm)
```
Comment (VourMa) [inline TTHAnalysis/python/plotter/mcPlots.py]: Do we need this comment after all? If yes, we should also notify Marco.

File: TTHAnalysis/python/plotter/susy-sos-v2-clean/mca-includes/2018/3l_semiddbkg/mca-data.txt [hunk 2]
```diff
@@ -1,9 +0,0 @@
-### 2018
```
Comment (VourMa) [inline TTHAnalysis/python/plotter/susy-sos-v2-clean/mca-includes/2018/3l_semiddbkg/mca-data.txt:1]: We are deleting this again, although it is not deleted in the other years. Can you confirm that this is intentional?

File: TTHAnalysis/python/plotter/susy-sos-v2-clean/mca-includes/2018/3l_semiddbkg/mca-fakes-applData.txt [hunk 3]
```diff
@@ -1,11 +1,11 @@
 ### 2017
```
Comment (VourMa) [inline TTHAnalysis/python/plotter/susy-sos-v2-clean/mca-includes/2018/3l_semiddbkg/mca-fakes-applData.txt]: This file seems to be for 2017 but it is included in a 2018 file.

**Review by VourMa — CHANGES_REQUESTED** [review]

**Comment by ipapaver** [comment]
________________________________

File: TTHAnalysis/python/plotter/mcPlots.py (hunk 1 above)
Comment (VourMa) [reply TTHAnalysis/python/plotter/mcPlots.py]: We indeed need to comment out this: `outputDir.WriteTObject(signorm)`. Could you please revert to the commented out version? I will merge the PR after that. Thanks!

**Pull request merged**